CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -423,6 +427,7 @@ await Task.Run(() =>
             }
             catch (Exception ex)
             {
+                await SentryHelper.ExceptionHandlerAsync(ex, SentryHelper.ExceptionType.UnhandledOther);
                 LogWriteLine($"Error has occurred while performing delta-patch!\r\n{ex}", LogType.Error, true);
                 throw;
```

Comment:
This catch might also throw ``OperationCancelledException`` or ``TaskCancelledException``. Type check is needed to exclude these two cancellation exceptions from sentry